### PR TITLE
Edited Red Hat info, added Xi'an

### PR DESCRIPTION
**Pull request by JunqiZhang0**
Edited Red Hat info, added Xi'an, we also have Xi'an Office in China

### CODE REVIEW

**Comment by formulahendry** [comment]
Please update https://github.com/formulahendry/955.WLB/blob/master/README_en_US.md as well

**Comment by JunqiZhang0** [comment]
@formulahendry Thanks! Done

File: README_en_US.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ The list of companies below, which are basically not affiliated with 996, is rel
 * PayPal - Shanghai
 * Pivotal - Beijing/Shanghai
 * Qualcomm - Beijing/Shanghai
-* Red Hat - Beijing/Shanghai/Shenzhen
+* Red Hat - Beijing/Shanghai/Shenzhen/Xi'an
```
Comment (formulahendry) [inline README_en_US.md:73]: To keep consistent, add 'remote' as well

Comment (JunqiZhang0) [reply README_en_US.md]: Done

**Review by formulahendry — APPROVED** [review]

**Pull request merged**